### ISSUE TITLE
Not working on Maya 2018

### ISSUE THREAD

**Issue by Skrytis** [giuliom95/arduino-maya#2]
Great work! We tried to build everything for windows 8 and maya 2018 with the very same code. We modified the serial2maya to work with windows, and everything seems fine. But nothing happens in the scene when moving the levers. Do you need other files in maya before you get this to work or?

**Comment by Skrytis** [comment]
arduino-maya-master\src\driver\serial2maya.py", line 44, in <module>
    cmd = 'arduinoUpdateChannel ' + str(i) + ' ' + v
TypeError: must be str, not bytes

So there's something wrong with the code...

**Comment by giuliom95** [comment]
Are you using Python 3, right?
If that so, try the fix I committed on the [issue/#2](https://github.com/giuliom95/arduino-maya/tree/issue/%232) branch.

**Comment by Skrytis** [comment]
Yes, I'm using Python 3.6.4. This is with the fix. Basically the same error but in a different place?
![image](https://user-images.githubusercontent.com/36663028/36505348-7ea788f2-1753-11e8-8b72-c1c37f210b0c.png)

**Comment by Skrytis** [comment]
I changed mayaconn.send(cmd) to mayaconn.send(bytes(cmd,'utf8')) and now it works! Really cool and thank you for this project!

**Comment by giuliom95** [comment]
Sorry I haven't tested it. Thanks for the fix. Are you going to make a pull request?
